ci-operator/step-registry/redhat-developer/rhdh/helm-aks-nightly-mapt-destroy/redhat-developer-rhdh-helm-aks-nightly-mapt-destroy-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
```

Comment:
```suggestion
set -e
```
So it can report failure in case its not able to destroy.